### PR TITLE
more restrictive cors

### PR DESCRIPTION
**Pull request by bersling**

### CODE REVIEW

File: backend/properties/README.md [hunk 1]
```diff
@@ -1,30 +0,0 @@
-# Configuration properties
```
Comment (AngelMunoz) [inline backend/properties/README.md:1]: I think this section could come handy on the backend readme.md

**Review by AngelMunoz — APPROVED** [review]
everything seems to check just a note about the properties, an explanation of why are there could come handy for some people but that could be explained in other ways (docs), and cors is always good to have

File: backend/properties/README.md (hunk 1 above)
Comment (bersling) [reply backend/properties/README.md:1]: Okay, I brought back the readme and improved it https://github.com/tsmean/tsmean/tree/fix/cors/backend/properties

**Review by AngelMunoz — APPROVED** [review]

File: backend/properties/README.md (hunk 1 above)
Comment (AngelMunoz) [reply backend/properties/README.md:1]: cool thanks

**Pull request merged**